content/upgrades/index.md

```diff
@@ -0,0 +1,436 @@
+
+Develop a well-documented process for handling cluster upgrades. Build runbooks and tooling to support cluster upgrades. 
+
+Plan to regularly upgrade your cluster. To keep up with EKS Kubernetes releases, upgrade clusters at least once per year. 
+
+## Review the EKS release calendar
+
+[Review the EKS Kubernetes release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar) to learn when new versions are coming, and when support for specific versions end. Generally, EKS releases three minor versions of Kubernetes annually, and each minor version is supported for about 14 months. 
+
+Additionally, review the upstream [Kubernetes release information](https://kubernetes.io/releases/).
+
+## Understand how the shared responsibility model applies to cluster upgrades
+
+You must initiate cluster upgrades. [Learn how to initiate an upgrade.](https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html) When you initiate a cluster upgrade, AWS manages upgrading the cluster control plane. You are responsible for upgrading the data plane, including [add-ons.](#upgrade-add-ons-and-components-using-the-kubernetes-api) You need to take additional action to upgrade the data plane, including Managed Node Groups, Fargate pods, and EKS Add-ons. You must validate and plan upgrades for workloads, controllers, and add-ons. 
+
+## Upgrade clusters in-place
+
+EKS supports an in-place cluster upgrade strategy. This will keep cluster configuration consistent (e.g., API endpoint, OIDC, ENIs, load balancers) which is less disruptive for cluster users, and it will use the existing workloads and resources in the cluster without requiring you to redeploy workloads or migrate external resources (e.g., DNS, storage).
+
+When performing an in-place cluster upgrade, it is important to note that only one minor version upgrade can be executed at a time (e.g., from 1.24 to 1.25). 
+
+This means that if you need to update multiple versions, a series of sequential upgrades will be required. Planning sequential upgrades is more complicated, and has a higher risk of downtime. In this situation, [evaluate a blue/green cluster upgrade strategy.](#evaluate-bluegreen-clusters-as-an-alternative-to-in-place-cluster-upgrades)
+
+## Upgrade your control plane and data plane in sequence
+
+To upgrade a cluster you will need to take the following actions:
+
+1. [Review the Kubernetes and EKS release notes.](#use-the-eks-documentation-to-create-an-upgrade-checklist)
+2. [Take a backup of the cluster. (optional)](#backup-the-cluster-before-upgrading)
+3. [Review add-on compatibility.](#upgrade-add-ons-and-components-using-the-kubernetes-api) Upgrade your Kubernetes add-ons and custom controllers, as required. 
+4. [Identify and remediate deprecated and removed API usage in your workloads.](#identify-and-remediate-removed-api-usage-before-upgrading-the-control-plane)
```

Comment:
Can you also make sure you have sort of similar ordering below as well. For eg taking backup section before upgrading the addon etc